### ISSUE TITLE
Onboard @mattbk

### ISSUE THREAD

**Issue by mattbk** [liberapay/salon#170]
If Gratipay shuts down, I'm potentially interested in helping out over here, depending on what needs to be done. I was primarily doing customer support and project review (I know the latter isn't needed here), with some small feature development.

**Comment by Changaco** [comment]
Well, there's plenty to do. If you're interested in continuing user support we can set it up (#11). As you said we don't do upfront reviews, but we still have to look out for fraud and general bad behaviour, and we need to progress in this area (clear policies, a proper dashboard for admins/moderators). Simple participation on GitHub is also good, I usually don't get any pull request reviews for example.

**Comment by mattbk** [comment]
Sounds good. I'm going to wait until https://github.com/gratipay/inside.gratipay.com/issues/1196 gets sorted out before I potentially jump into something new.

**Comment by mattbk** [comment]
I should be able to help with support if #11 is set up. What volume of support emails is Liberapay receiving? I'm not a whiz with languages but I can triage using Google Translate.

The reason I got into support with Gratipay is that I was annoyed with how things weren't working at the time, so I started reading the GitHub queue. Then other people would show up with questions and I had read enough to figure out the answers. So it may take me a little time to get up to speed in the differences (as you can tell by the few tickets I've opened so far).

**Comment by Changaco** [comment]
> What volume of support emails is Liberapay receiving?

I handled 6 requests in October. That's 8 responses sent to users, and 3 inquiries sent to Mangopay to figure out a payment failure.

> The reason I got into support with Gratipay is that I was annoyed with how things weren't working at the time, so I started reading the GitHub queue. Then other people would show up with questions and I had read enough to figure out the answers.

When possible I fix an issue after receiving a first or second email about it, so that I won't have to answer the same question over and over again.

**Comment by mattbk** [comment]
Sounds like you don't need me then.

**Comment by Changaco** [comment]
Having more people to handle support may not be necessary (yet?), but it would be appreciated nonetheless. Every email is a distraction.

I've been looking at #11, I'll have more to say on that soon.

**Comment by Changaco** [comment]
@mattbk Why did you leave? 😢

**Comment by mattbk** [comment]
Trying to prioritize my time and I'm not very effective here.

**Comment by Changaco** [comment]
Okay. Sorry I failed to help you become "effective".

(BTW I've received or sent 159 support emails so far in December.)

**Comment by mattbk** [comment]
As you've noticed, I have a little more bandwidth now, but probably not enough to be responsible for anything. Still happy to fiddle with shields.io as needed.

**Comment by mattbk** [comment]
I humbly request to be re-added to the team.

**Comment by Changaco** [comment]
Invite sent. Welcome back! :heart:

**Comment by mattbk** [comment]
Can I be added to the GitHub team so I can tag/reopen/close issues?

I'd also be interested in higher privileges on liberapay.com to be more effective at helping people figure out "what happened with my money" questions. I was an admin on Gratipay for this reason. You can ask Chad for a reference if needed. If that's not cool, no worries.

**Comment by Changaco** [comment]
I trust you @mattbk, but I don't want to give the admin status to anyone else until I've modified it to be less dangerous.

Before giving you privileges that will allow you to access private data I also have to ask you to confirm that your Liberapay and email accounts are properly secured (by strong, preferably random, passwords).

**Comment by mattbk** [comment]
>  I don't want to give the admin status to anyone else until I've modified it to be less dangerous.

I agree with that! I don't want to break anything either.

>  I also have to ask you to confirm that your Liberapay and email accounts are properly secured (by strong, preferably random, passwords).

Confirmed.

**Comment by Changaco** [comment]
> I don't want to give the admin status to anyone else until I've modified it to be less dangerous.

For the record, https://github.com/liberapay/liberapay.com/pull/1373 was the first step towards making the admin privilege less "dangerous". The second step will be to rate-limit admin actions. I'm also planning to review the actions that admins can perform, looking for anything that could be problematic.

**Comment by mattbk** [comment]
May I be added to the GitHub org so I can make branches directly?

**Comment by Changaco** [comment]
I thought you already were, sorry. I've added you to the Techs team.

**Comment by Changaco** [comment]
I've also granted you the admin privilege on Liberapay. I still haven't done my review of what admins can modify and delete, so be careful.

**Comment by mattbk** [comment]
Would you like any help reviewing accounts? We can discuss over email if you prefer not to disclose your methods.

**Comment by Changaco** [comment]
@mattbk Yes, help welcome. When reviewing accounts I look at both the donors and the recipient to try to determine if it's plausible that the donations are legit. When it's not clear I leave the account "unreviewed".

**Comment by mattbk** [comment]
Anything to be done about accounts like this? https://liberapay.com/for/RlVDSyBZT1U

**Comment by Changaco** [comment]
Yes, you can switch on `hide_from_lists` and `hide_from_search` for this kind of fake account.

**Comment by mattbk** [comment]
Is there a point at which you switch `profile_noindex (user value: False) ` to `False`?

(Edit: Messed up my booleans.)

**Comment by Changaco** [comment]
`profile_noindex` is on by default, I set it to `False` if the profile looks legit and uncontroversial.
